internal/devices/update.go

```diff
@@ -62,6 +64,18 @@ func (c *Client) Update() *cobra.Command {
 				deviceUpdate.Description = &description
 			}
 
+			if userdata != "" && userdataFile != "" {
+				return fmt.Errorf("either userdata or userdata-file should be set")
```

Comment:
This change would require userdata changes whenever updated a device. 

Want to change the hostname? Include userdata, please.

Reply:
PR otherwise looks good - just remove this condition block.

Reply:
This won't force userdata changes; it's validating that the user only specified one of `--userdata <some_string>` or `--userdata-file <some_file>`.